Getting started docs (#27)
Qowaiv/qowaiv-js: GETTING_STARTED.md

# Getting Started

> This guide will setup your local environment.

Reviewer: ```suggestion
This guide will set up your local environment.
```

## Prerequisite

> You are running on linux or running in a linux wsl environment.

Reviewer 2: ```suggestion
You are running on Linux or running in a [Linux WSL](https://learn.microsoft.com/windows/wsl/install) environment.
```

> To setup your environment we are depending on devbox to have an isolated environment and all required tools are pre installed

Reviewer 2: ```suggestion
To setup your environment we are depending on [MS Dev Box(](https://azure.microsoft.com/en-us/products/dev-box). This ensures an isolated environment and all required tools to be installed.
```

Reviewer: setup => set up

> To install devbox open a terminal and run the following command

Reviewer 2: ```suggestion
To install DevBox open a terminal and run the following command:
```

> ```bash

Reviewer 2: ```suggestion
``` bash
```

> To setup your environment we are depending on [MS Dev Box(](https://azure.microsoft.com/en-us/products/dev-box). This ensures an isolated environment and all required tools to be installed.

Reviewer 2: ```suggestion
To set up your environment we are depending on [MS Dev Box(](https://azure.microsoft.com/en-us/products/dev-box). This ensures an isolated environment and all required tools to be installed.
```

curl -fsSL https://get.jetify.com/devbox | bash
```

## Setup Environment

To open the environment run the following command

> ```bash

Reviewer 2: ```suggestion
``` bash
```

devbox shell
```

this will install all the required tools in the environment.

> open the vscode

Reviewer 2: ```suggestion
open VSCode:
```
